config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  post '/graphql', to: 'graphql#query'
+  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: '/graphql'
```

Comment:
Ideally, at the end of this project we'd be able to do this with our own engine. Then see a tool like graphiql except instead of the graphql response we'd visualize the analyzed query.

```ruby
mount GraphQL::Analyzer::Engine, at: "/graphql-analyze", graphql_analyze_path: '/graphql/analyze'
```